### ISSUE TITLE
Override CTRL + K to Open Search Box

### ISSUE THREAD

**Issue by omjogani** [opxica/modular-ui-website#14]
In most of the documentation there is overridden shortcut <mark>CTRL + k</mark> that directly open search window to search it directly. The same is also required at docs page.

example: [Tailwind Docs](https://tailwindcss.com/docs/installation)

Thanks!

**Comment by omjogani** [comment]
@n-georges You may take over while you're working on it!

**Comment by VishruthVS** [comment]
@omjogani I think it has already been implemented !

**Comment by omjogani** [comment]
> @omjogani I think it has already been implemented !

No, It's not implemented. On pressing Ctrl/Cmd + K it should open search box in center of the screen just like you click on search button.

![image](https://github.com/opxica/modular-ui-website/assets/72139914/761c1756-f96c-4594-870d-44acbd4f05f5)
